app/InterceptAllButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+export const InterceptAllButton = props => {
```

Comment:
This component doesn't need to know about `checkedReqs` or `enabledRequests`.
It should accept a eventHandler and `disabled` props